### DISCUSSION TITLE
When did you think you could actually play Switch game on the Skyline emulator?

### DISCUSSION THREAD

**Discussion by Paskerprima** [skyline-emu/skyline#820] [Q&A] [answered]
When did you think you could actually play Switch game on the Skyline emulator?

**Comment by yxles-dev** [comment]
You already can tho?

**Reply by Paskerprima** [reply]
I mean besides celeste, and sonic mania

**Reply by KristalFragmentz** [reply]
I'd say, considering the advancing development speed, a year at least.

**Comment by Crytonics** [comment, marked as answer]
You can now but not all games ;)
